### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by CoreyHorn**
Add connect function to interactors to ensure
Presenter is connected before pushing results.
Update gitignore

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (approxinfinity) [inline .idea/misc.xml]: do we want this file in .gitignore as well?

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (approxinfinity) [inline .idea/vcs.xml]: do we want this file in .gitignore as well?

File: build.gradle [hunk 3]
```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
+        classpath 'com.android.tools.build:gradle:3.1.0-alpha08'
```
Comment (approxinfinity) [inline build.gradle:11]: we good with alphas? don't think we ever talked about policy on this.

File: mvpiframework/src/main/java/com/coreyhorn/mvpiframework/MVPISettings.kt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.coreyhorn.mvpiframework
+
+object MVPISettings {
```
Comment (approxinfinity) [inline mvpiframework/src/main/java/com/coreyhorn/mvpiframework/MVPISettings.kt:3]: should this sit in app? Is it accessible from app to set it?

**Review by approxinfinity — CHANGES_REQUESTED** [review]
nice. couple things.

File: mvpiframework/src/main/java/com/coreyhorn/mvpiframework/MVPISettings.kt (hunk 4 above)
Comment (CoreyHorn) [reply mvpiframework/src/main/java/com/coreyhorn/mvpiframework/MVPISettings.kt:3]: I believe it should be accessible from the app.

File: build.gradle (hunk 3 above)
Comment (CoreyHorn) [reply build.gradle:11]: Actually just let Android Studio update that. I'm fine with it for build tools but either way works for me.

File: .idea/vcs.xml (hunk 2 above)
Comment (CoreyHorn) [reply .idea/vcs.xml]: We should pull both of these, good catch.

File: build.gradle (hunk 3 above)
Comment (approxinfinity) [reply build.gradle:11]: I'm in stable channel. I think I'm good with whatever auto updates via stable... think that might be best policy? :man_shrugging:

**Pull request merged**